.github/workflows/rust.yml

```diff
@@ -50,7 +50,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust: [nightly-2020-11-24]
```

Comment:
I know this is still in-flight, but the better strategy here is to run the Parquet tests with `cargo +stable test` as we also add the stable toolchain to compile arrow with. You might need to rebase to see that in the dev scripts, as this change was added last week.

Reply:
I am ok with that, I am fighting the CI anyhow and any assistance is welcome.

Reply:
I'll submit a PR on your branch

Reply:
I think the safest option is to revert the CI changes, then we can add parquet to stable tests separately in #8821

Reply:
changes reverted, I will probably need help getting the small CI parts fixed up